public/lang/babele/de/investigator.pathOfCthulhuAbilities.json

```diff
@@ -178,14 +178,14 @@
         },
         "Oral History": {
             "name": "Zeugenbefragung",
```

Comment:
I switched this to "Mündliche Geschichte" a while ago, which is a quiet literal translation as Zeugenbefragung does not really cover the historical aspect of this skill.